presto-benchto-benchmarks/src/test/resources/sql/presto/tpcds/q09.plan.txt

```diff
@@ -13,110 +13,81 @@ remote exchange (GATHER, SINGLE, [])
                                                 cross join:
                                                     cross join:
                                                         cross join:
-                                                            cross join:
-                                                                scan reason
```

Comment:
what is the reason it changed?

Reply:
The immediate reason is that an output column was pruned of the `reason` table (due to the introduced project-off rule). As a consequence, `DetermineJoinDistributionType` decided to flip Join sources.